notebooks/IMPACT_SERVIR_HLS_FM_Customer_Container_G4G.ipynb

```diff
@@ -78,10 +87,10 @@
         "\n",
         "The steps performed include:\n",
         "\n",
-        "- FastAPI server to handle predictions and health checks.\n",
-        "- Push GCP cloud build to build and push artifacts to GCP Artifcat Registry.\n",
-        "- Create Vertex AI Model from the custom image in Artifact Registry.\n",
-        "- Deploy the model to Vertex AI Endpoint for online prediction. "
+        "- Get the custom code ready with Docker container to be used in the Google's cloud run.\n",
+        "- Build a FastAPI server to handle predictions and health checks.\n",
+        "- Build a custom container with model artifacts.\n",
+        "- Upload and deploy custom container to Vertex AI Endpoints."
```

Comment:
Use the following list:

"- FastAPI server to handle predictions and health checks.\n",
"- Push GCP cloud build to build and push artifacts to GCP Artifcat Registry.\n",
"- Create Vertex AI Model from the custom image in Artifact Registry.\n",
"- Deploy the model to Vertex AI Endpoint for online prediction. "